config/attachments.php

```diff
@@ -15,6 +15,9 @@
         'prefix' => 'attachments',
         'middleware' => 'web',
         'pattern' => '/{id}/{name}',
+        'sharing' => [
```

Comment:
maybe directly `share_pattern` instead of array ? The route definition must be updated according